ios/Video/RCTVideo.swift

```diff
@@ -193,11 +193,11 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
     }
 
     @objc func applicationDidEnterBackground(notification:NSNotification!) {
-        if !_playInBackground {
-            // Needed to play sound in background. See https://developer.apple.com/library/ios/qa/qa1668/_index.html
-            _playerLayer?.player = nil
-            _playerViewController?.player = nil
-        }
+       if _playInBackground {
```

Comment:
I don't believe that is the correct solution as it will disrupt `playInBackground`.
Pls try to find the right solution here: https://github.com/TheWidlarzGroup/react-native-video/issues/2118#issuecomment-1444988965

Reply:
Hi Jason,

This was the analysis done

Seems like in react-native-video we have picked up all the alpha-8 fixes and merged but we are still using alpha-7 so many functionality are breaking in IOS.
For, PictureInPicture I see there was a issue reported in aplha-8 - https://github.com/TheWidlarzGroup/react-native-video/issues/3271
and https://github.com/TheWidlarzGroup/react-native-video/pull/3221/files was the solution. But we have this code in our aplha-7.
After correcting the below change, PIP is working now.
@objc func applicationDidEnterBackground(notification:NSNotification!) {
        if _playInBackground {
            // Needed to play sound in background. See https://developer.apple.com/library/ios/qa/qa1668/_index.html
            _playerLayer?.player = nil
            _playerViewController?.player = nil
        }
    }
In the above shared link many people are suggesting this fix as well.
https://github.com/TheWidlarzGroup/react-native-video/issues/2177#issuecomment-770474082

Reply:
Also since it is only patch fix what we are going to make since PIP is not implemented correctly as of now, Can we proceed with this since impact of changes will be only with small area ?

Reply:
In react-native-video we have picked up all the alpha-8 fixes and merged but we are still using alpha-7 so many functionality are breaking in IOS.
For, PictureInPicture I see there was a issue reported in aplha-8 - https://github.com/TheWidlarzGroup/react-native-video/issues/3271
and https://github.com/TheWidlarzGroup/react-native-video/pull/3221/files was the solution. But we have this code in our aplha-7.
After correcting the below change, PIP is working now.
@objc func applicationDidEnterBackground(notification:NSNotification!) {
        if _playInBackground {
            // Needed to play sound in background. See https://developer.apple.com/library/ios/qa/qa1668/_index.html
            _playerLayer?.player = nil
            _playerViewController?.player = nil
        }
    }

Reply:
The fix committed here is also working with playInBackground. However I am making the changes suggested. But this might add a lot of regression.